ui-speedspacechart/src/components/layers/ReticleLayer.tsx

```diff
@@ -24,6 +24,8 @@ const ReticleLayer = ({ width, height, heightOffset, store }: ReticleLayerProps)
     if (store.cursor.y && store.cursor.y < maxCursorHeight) {
       const detailsBox = drawCursor({ ctx, width, height, store });
       setTrainDetails(detailsBox || null);
+    } else {
+      setTrainDetails(null);
     }
```

Comment:
You can also clear th reticle using the `clearCanvas` function that you can import from `../utils` too
```suggestion
    } else {
      clearCanvas(ctx, width, height);
      setTrainDetails(null);
    }
```

Reply:
thanks 👍